src/axios.ts

```diff
@@ -3,12 +3,19 @@ import http from 'http';
 import https from 'https';
 import emitter from './emitter';
 
+// For a compatibility with Snyk CLI
+export declare interface Global extends NodeJS.Global {
+  ignoreUnknownCA: boolean;
```

Comment:
Required: Does CA stand for certification Authorities? Could we add a comment explaining that?

Reply:
`CA` should be widely known abbrev, so I don't wanna to add a comment just because of that. However, I can elaborate on a comment explaining, how it can be turned on.